### ISSUE TITLE
usage(): Split SYNOPSIS and DESCRIPTION

### ISSUE THREAD

**Issue by Lestropie** [MRtrix3/mrtrix3#904]
Pretty much every command starts the `DESCRIPTION` field with a single sentence describing the purpose of the command; some then go on to give further details. If this were instead split into two fields, e.g. `SYNOPSIS` for the first line and `DESCRIPTION` for anything subsequent to that, then a functionality like `__print_synopsis__` could be added, and used to populate the [command list documentation page](http://mrtrix.readthedocs.io/en/latest/reference/commands_list.html) with a small amount of information on each command.

**Comment by draffelt** [comment]
Wound be very useful

**Comment by jdtournier** [comment]
Had a go at this with while I was hacking away at  #932 - see #933. I didn't add an explicit `SYNOPSIS` field, but decided to simply take the first entry in the description to populate the list. We just need to communicate to app developers that this first line needs to be kept short and punchy. You may want to tweak the scripts' handling of the description field in some way to match, but currently none of the descriptions are that long anyway...

**Comment by thijsdhollander** [comment]
This should probably have been closed upon [this commit](https://github.com/MRtrix3/mrtrix3/commit/a12939fff61a84cd63d1e0a6558778eb391aee0f) (see above).  But there's a little :information_source: icon next to the commit's listing in this issue that says "This issue will close once commit a12939f is merged into the 'master' branch.".

Long story short: we need to close issues that get addressed by a commit that doesn't immediately end up on `master` (e.g. all work on `tag_0.3.16`) manually...

**Comment by thijsdhollander** [comment]
(the :information_source: icon is gone now that I have closed the issue manually)
